program/src/processor.rs

```diff
@@ -39,6 +39,7 @@ fn calculate_proposal_vote_threshold(stake: u64, total_stake: u64) -> Result<u64
     stake
         .checked_mul(THRESHOLD_SCALING_FACTOR)
```

Comment:
this can be done separately, but I think we'll need to widen for this calculation -- with u64s, and a token with 9 decimal places (like SOL), it means that a vote could only handle 18 SOL max :sweat_smile:

Reply:
You're saying widen the scaling out, by converting to `u128` as we do in rewards?

Reply:
Yeah, just during this function, no need to store the wider value